docs/model_developers/handlers.rst

```diff
@@ -233,28 +233,28 @@ method annotated with this annotation will be cached, similar to the way
 `lru_cache <https://docs.python.org/3/library/functools.html#functools.lru_cache>`_ works. The arguments to
 the method will form the cache key, the return value will be cached. When the method is called a
 second time with the same arguments, it will not be executed again, but the cached result is
-returned instead. To exclude specific arguments from the cache key, use the `ignore` parameter.
+returned instead. To exclude specific arguments from the cache key, use the ``ignore`` parameter.
 
-Cache entries will be dropped from the cache when they become stale.
-The for_version and timeout parameters control this behaviour:
+Cache entries will be dropped from the cache when they become stale. Use the following parameters to control when an entry is considered stale:
+  * Setting ``evict_after_creation=True`` will mark entries as stale after a timeout controlled by the ``timeout`` parameter (5000s by default).
+  * Setting ``refresh_after_access=True`` will reset the expiry time of entries to 60s anytime they're used.
```

Comment:
I am not really a big fan of changing this from `evict_after_last_access` to `refresh_after_access`. The term "evict" makes it really clear this is about cache invalidation and the term "after_last_access" makes it clear the we will evict after the last access. `refresh_after_access` is a lot more vague. Refresh can mean a lot of different things. I also don't really like that we had a naming schema for the variables `evict_<something>`. That's something we lost now.

Reply:
I agree with this. As originally thought, the renaming I proposed was specific to the `CacheItem` class because checking: 
```
if item.evict_after_last_access
```
to see if an item needs to be refreshed is not so clear to me. But maybe I am the only one bothered by this

Reply:
I agree, I'll revert this and only apply the renaming to the `CacheItem` class.